app/initgraphql.go

```diff
@@ -19,6 +19,7 @@ func (app *App) initGraphQL(ctx context.Context) error {
 
 	app.graphql2 = &graphqlapp.App{
 		DB:                app.db,
+		AuthBasicStore:    *app.AuthBasicStore,
```

Comment:
Change `AuthBasicStore` to a pointer type since `basic.Store` is not an interface, rather than dereferencing it.